test/googleSheets.test.ts

```
@@ -203,84 +232,180 @@ describe('Google Sheets Integration', () => {
+        { cols: 52, expected: 'AZ' },
+        { cols: 53, expected: 'BA' },
+        { cols: 702, expected: 'ZZ' },
+        { cols: 703, expected: 'AAA' },
+      ];
+
+      for (const { cols, expected } of testCases) {
+        jest.clearAllMocks(); // Clear mocks between iterations
+
+        const headers = Array.from({ length: cols }, (_, i) => `col${i + 1}`);
+        const mockRows: CsvRow[] = [
+          headers.reduce((acc, header) => ({ ...acc, [header]: 'value' }), {}),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)